docs-kits/kits/knowledge-agents/development-view/modelling.md

```diff
@@ -0,0 +1,714 @@
+---
+sidebar_position: 1
+title: Ontology Modelling
+---
+<!--
+ * Copyright (c) 2024 T-Systems International GmbH
+ * Copyright (c) 2024 Mercedes-Benz AG
+ * Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This documentation and the accompanying materials are made available under the
+ * terms of the Creative Commons Attribution 4.0 International License,  which is available at
+ * https://creativecommons.org/licenses/by/4.0/legalcode.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: CC-BY-4.0
+-->
+
+![Agents Kit Banner](/img/knowledge-agents/AgentsKit-Icon.png)
+
+## Agents KIT
+
+**Ontology models for federated queries over the whole data space.**
+
+For more information see
+
+* Our [Adoption](../adoption-view/intro) guideline
+* The [High-Level Architecture](architecture)
+* The [ARC42](Arc42) documentation
+* An [API](api) specification
+* The [Layers & Modules](modules) Architecture
+* Our [Reference Implementation](reference)
+* The [Deployment](../operation-view/deployment) guide
+
+## Introduction & Standards
+
+This document gives fundamental guidelines to create ontology models for being used in Catena-X (see Catena-X standard CX-0067). It provides the semantic basis for the so called Knowledge Agent Approach that utilizes semantic web technologies for federated queries in data spaces (see Catena-X standard CX-0084).
+
+* [CX-0084 Federated Queries in Dataspaces (V1.2.0)](https://catena-x.net/de/standard-library/Update_Mai_2024/CX-0084-FederatedQueriesInDataSpaces-v1.2.0.pdf)
+* [CX-0067 Ontology Models to Realize Federated Queries in Catena-X (V1.1.0)](https://catena-x.net/de/standard-library/Update_Mai_2024/CX-0067-OntologyModelsinCatenaX-v1.1.0.pdf)
+
```

Comment:
As the URLs to the PDF-files won't work anymore after the Release 24.05, I recommend to update the URL to: https://catena-x.net/de/standard-library